string direction  # Desired rollerbrush direction (up/down)
---
bool success   # Response if direction is successfully set
string message # Response message
